crates/bevy_render/src/camera/bundle.rs

```diff
@@ -75,6 +75,23 @@ pub struct OrthographicCameraBundle {
 }
 
 impl OrthographicCameraBundle {
+    /// Create an orthographic projection camera to render 2D content.
+    ///
+    /// The projection creates a camera space where X points to the right of the screen,
+    /// Y points to the top of the screen, and Z points out of the screen (backward),
+    /// forming a right-handed coordinate system. The center of the screen is at `X=0` and
+    /// `Y=0`.
+    ///
+    /// The default scaling mode is [`ScalingMode::WindowSize`], resulting in a resolution
+    /// where 1 unit in X and Y in camera space corresponds to 1 pixel on the screen. That
+    /// is, for a screen of 1920 pixels in width, the X coordinates visible on screen go
```

Comment:
Is it interesting to mention that "pixels" is logical pixels? More and more screen have a scale factor

Reply:
Added.